2_process/src/process_dv_historic_quantiles.R

```diff
@@ -0,0 +1,32 @@
+#' @title Calculate the stat category for each gage's discharge value
+#'
+#' @param ind_file character file name where the output should be saved
+#' @param dv_historic_ind indicator file for the data.frame of dv_historic tmp files
+#' @param percentiles character vector of the types of stats to include, i.e. `c("10", "75")`
+#' will return the 10th and 75th percentiles (from viz_config.yml)
+process_dv_historic_quantiles <- function(ind_file, dv_historic_ind, percentiles){
+
+  dv_historic_data <- do.call(rbind, lapply(names(yaml.load_file(dv_historic_ind)), readRDS))
```

Comment:
I thought you were worried this would be too big? If it turns out to be so, you could read just one file at a time, summarize that file, save the summary, then overwrite the data with the data from the next file.

Reply:
Oh right, good point. Luckily, I only kicked off the fetch part, so I can still edit this! I'll switch it around based on your suggestion.